build_tools/third_party/swiftshader/build_vk_swiftshader.sh

```diff
@@ -54,7 +54,7 @@ else
   SWIFTSHADER_INSTALL_DIR="${BASE_DIR?}"'\.swiftshader'
 fi
 
-SWIFTSHADER_COMMIT=c27e99245d423648fe5f0eafa73d49ce7b53d201
+SWIFTSHADER_COMMIT=57eca9df5dc56d290f662ace7481ed758618d559
```

Comment:
This should be (ideally) kept in sync with https://github.com/google/iree/blob/47e45d3c433a4b20eb9ef28c2340e47ae7108ca0/build_tools/docker/swiftshader/Dockerfile#L12 which requires rebuilding the Docker images.

See https://github.com/google/iree/commit/dd1d96c8feffd693d64a73436d60d249570b8be5 for a full commit bumping SwiftShader

Reply:
Should be fixed.